sdk/formrecognizer/Azure.AI.FormRecognizer/samples/Sample4_StronglyTypingARecognizedForm.md

```diff
@@ -0,0 +1,79 @@
+# Strongly-typing a recognized form
+
+This sample demonstrates how to use the fields in your recognized forms to create an object with strongly-typed fields. The pre-trained receipt model will be used to illustrate this sample, but note that a similar approach can be used for any custom form as long as you properly update the fields' names and types.
+
+To get started you'll need a Cognitive Services resource or a Form Recognizer resource.  See [README][README] for prerequisites and instructions.
+
+## Creating a `FormRecognizerClient`
+
+To create a new `FormRecognizerClient` you need the endpoint and credentials from your resource. In the sample below you'll use a Form Recognizer API key credential by creating an `AzureKeyCredential` object, that if needed, will allow you to update the API key without creating a new client.
+
+You can set `endpoint` and `apiKey` based on an environment variable, a configuration setting, or any way that works for your application.
+
+```C# Snippet:CreateFormRecognizerClient
+string endpoint = "<endpoint>";
+string apiKey = "<apiKey>";
+var credential = new AzureKeyCredential(apiKey);
+var client = new FormRecognizerClient(new Uri(endpoint), credential);
+```
+
+## Strongly-typing a receipt
```

Comment:
Should we add an "overview" section that describes the high-level steps for doing this?
- Create a wrapper class using `FormField<T>` for the known properties
- Implement a constructor that takes `RecognizedForm`
- Convert from fields in `RecognizedForm` to the strongly-typed properties

I personally benefit from knowing at a high-level what's going to happen before I start diving into the code.

Reply:
I separated it into two sections: `Creating a wrapper class` and `Accessing fields in the strongly-typed receipt`. I added more info before presenting the first snippet, and included a snippet to the `Receipt` constructor.